src/uris/deepLinks/deepLink.ts

```diff
@@ -107,9 +111,22 @@ export const enum DeepLinkServiceAction {
 	TargetFetched,
 }
 
+export enum DeepLinkRepoOpenAction {
```

Comment:
We shouldn't use `enum`s for UI -- as localization would break this

Reply:
Done in commit: [Improvements from feedback](https://github.com/gitkraken/vscode-gitlens/pull/2503/commits/9e01ae826ec49b794057aa9570ec3df54ef0042d)